shelley/chain-and-ledger/shelley-spec-ledger-test/src/Test/Shelley/Spec/Ledger/Generator/Utxo.hs

```diff
@@ -368,6 +373,7 @@ genNextDelta
         draftSize =
           sum
             [ 5 :: Integer, -- safety net in case the coin or a list prefix rolls over into a larger encoding
+              12 :: Integer, -- TODO the size calculation somehow needs extra buffer when minting tokens
```

Comment:
I do not understand why this was needed, but this turned out to be the smallest change that I could make to get the fee calculation to cover the minimal fee.